lms/templates/verify_student/image_input.underscore

```diff
@@ -0,0 +1,2 @@
+<img class="preview" alt="<%- gettext("Preview of uploaded image") %>"/>
+<input class="image-upload" type="file" accept="image/*;capture=camera">
```

Comment:
@wedaly thanks for pulling me in on this.  As with any other ```<input>``` element, we will need an accessible label that describes what this control is for.  The OS (mobile or otherwise) will inform the user that this is a file upload, but will most likely not tell them what the restrictions on file type will be.  You can present the text visually if you think it would help everyone, or you can display it offscreen with our standard ```sr``` class.  Here is an example:
Visible
```
<label>
  Upload an image or capture one with your web or phone camera
  <input class="image-upload" type="file" accept="image/*;capture=camera">
</label>
```
Hidden:
```
<label>
  <span class="sr">Upload an image or capture one with your web or phone camera</span>
  <input class="image-upload" type="file" accept="image/*;capture=camera">
</label>
```